x/ccv/provider/keeper/punish_validator.go

```diff
@@ -3,26 +3,24 @@ package keeper
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	evidencetypes "github.com/cosmos/cosmos-sdk/x/evidence/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
 	"github.com/cosmos/interchain-security/v2/x/ccv/provider/types"
 )
 
-// JailValidator jails the validator with the given provider consensus address
-// Note that the tombstoning is temporarily removed until we slash validator
-// for double signing on a consumer chain, see comment
-// https://github.com/cosmos/interchain-security/pull/1232#issuecomment-1693127641.
-func (k Keeper) JailValidator(ctx sdk.Context, providerAddr types.ProviderConsAddress) {
+// JailAndTombstoneValidator jails and tombstones the validator with the given provider consensus address
+func (k Keeper) JailAndTombstoneValidator(ctx sdk.Context, providerAddr types.ProviderConsAddress) {
```

Comment:
Renamed to `JailAndTombstoneValidator` because this was what this method was doing [initially](https://github.com/cosmos/interchain-security/commit/f168b9b4a5d15f3641d9cb09182b923f7688aead). However, I intend to break it into 2 methods: `JailValidator` and `TombstoneValidator`.